### PR TITLE
Configured links to enforce openjdk8  repo

### PR DESCRIPTION
**Pull request by EnriqueL8**
[ON STAGING](https://staging.adoptopenjdk.net) 
Setting archive and nightly build links to point to openjdk8 repositories. 

##### Checklist

- [x] `npm test` passes
- [ ] documentation is changed or added
- [X] contribution guidelines followed [here](https://github.com/AdoptOpenJDK/openjdk-website/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: src/handlebars/partials/menu.handlebars [hunk 1]
```diff
@@ -9,6 +9,7 @@
       <li><a href="./archive.html">Build archive</a></li>
       <li><a href="./nightly.html">Nightly builds</a></li>
       <li><a href="./installation.html">Installation</a></li>
+      <li><a href="http://blog.adoptopenjdk.net/">Blog</a></li>
```
Comment (karianna) [inline src/handlebars/partials/menu.handlebars]: https?

File: src/js/archive.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ function populateArchive() {
 
     // var repoName = (variant + '-releases');
 
-    loadJSON('openjdk-releases', 'releases', function(response) {
+    loadJSON('openjdk8-releases', 'releases', function(response) {
```
Comment (karianna) [inline src/js/archive.js]: OK we'll have to make this work for OpenJDK 9 as well at some point, but that's a later fix.

File: src/js/index.js [hunk 3]
```diff
@@ -52,55 +52,66 @@ function buildHomepageHTML(releasesJson) {
...
+      var uppercaseOSname = null;
       // firstly, check if a valid searchableName has been returned (i.e. the platform is recognised)...
       if(thisPlatform) {
 
         // secondly, check if the file has the expected file extension for that platform...
         // (this filters out all non-binary attachments, e.g. SHA checksums - these contain the platform name, but are not binaries)
-        var thisBinaryExtension = getBinaryExt(thisPlatform); // get the file extension associated with this platform
-        if(uppercaseFilename.indexOf(thisBinaryExtension.toUpperCase()) >= 0) {
-          var uppercaseOSname = OS.searchableName.toUpperCase();
+        var thisBinaryExtension = getBinaryExt(thisPlatform); // get the binary extension associated with this platform
```
Comment (karianna) [inline src/js/index.js]: Comments are probably not needed, function call is well named

File: src/js/archive.js (hunk 2 above)
Comment (Joe-Brady) [inline src/js/archive.js]: @karianna I have worked with Enrique just now to change this - I had set it up before I left to work with all variants that are defined inside `config.json`. So Enrique has now changed these lines of code to activate this system.

When OpenJDK 9 comes along, it's just a matter of choosing a naming convention for the new repositories, then adding a few lines to `config.json` to match this naming convention.

**Pull request closed without merging**